packages/Table/stories/Table.collections.stories.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { getStoryName } from "storybook/storyTree";
+import styled from "styled-components";
+
+import Button from "@paprika/button";
+import Pill from "@paprika/pill";
+import DropdownMenu from "@paprika/dropdown-menu";
+
+import Table from "../src";
+
+const storyName = getStoryName("Table");
+
+storiesOf(storyName, module);
+
+const data = [
+  {
+    label: "Questionnaire tablet that is very long",
+    type: "questionnaire",
+    description: "",
+    tag: "47,9000",
+    interpretations: 15,
+    triggers: 15,
+    metrics: 15,
+  },
+  {
+    label: "DataTable title 1",
+    type: "dataTable",
+    data: [{}, {}],
+    description: `This is a description that is part of the collection. 
+                  It only needs to show this much stuff in this area, it can go on u…`,
+    tag: "47,9000",
+    interpretations: 15,
+    triggers: 15,
+    metrics: 15,
+  },
+  {
+    label: "Completely empty table",
+    type: "dataTable",
+    description: "",
+    data: [],
+    interpretations: 15,
+    triggers: 15,
+    metrics: 15,
+  },
+  {
+    label: "Another data analytic but has the script name",
+    type: "dataTable",
+    script: "alert(void(0))",
+    description: "",
+    tag: "47,9000",
+    interpretations: 15,
+    triggers: 15,
+    metrics: 15,
+  },
+];
+
+const titleStyles = {
+  Title: styled.div`
+    display: flex;
+  `,
+  Label: styled.div`
+    flex-basis: 100%;
+  `,
+};
+
+function Title(props) {
+  const { label, description, tag } = props;
+
+  return (
+    <div>
+      <titleStyles.Title>
+        <titleStyles.Label>
+          <Button.Link size="big">{label}</Button.Link>
```

Comment:
Shouldn't `size` be a `CustomPropTypes.ShirtSize` value? Do we need to create a new one?

Reply:
I'm not fan of using micro abstraction, which adds unnecessary complexity IMHO while reading the code. A lot of my thoughts about this are part of the comment that I added about our Constant package :)

Reply:
Ok. I figured we'd standardized the approach but maybe not. I'll update the types/enums pr to try and close it out and solidify an approach.